scripts/updateRiskOracleConfig.ts

```diff
@@ -1,51 +1,85 @@
 import hre from "hardhat";
-
 import { encodeData } from "../utils/hash";
+import { getFullKey } from "../utils/config";
 import * as keys from "../utils/keys";
 
 export async function main() {
+  const riskOracleConfig = await hre.gmx.getRiskOracle();
+
+  const dataStore = await hre.ethers.getContract("DataStore");
   const config = await hre.ethers.getContract("Config");
 
-  const syncConfigKeys = [
-    // {
-    //   baseKey: keys.SYNC_CONFIG_MARKET_DISABLED,
-    //   data: encodeData(["address"], ["0x352f684ab9e97a6321a13CF03A61316B681D9fD2"]),
-    // },
-    {
+  if (!riskOracleConfig) {
+    throw new Error("No configuration found for the current network");
+  }
+
+  const syncConfigKeys = [];
+  
+  for (const marketAddress in riskOracleConfig.markets) {
```

Comment:
you can use `for (const [marketAddress, marketConfig] of Object.entries(riskOracleConfig.markets) { ... }`

it's better to not use `for in` to avoid unexpected behavior (for example, with arrays it will return both elements and some array properties)